tests/functionnal/Glpi/Inventory/Inventory.php

```diff
@@ -1779,11 +1779,12 @@ public function testUpdateComputer()
             'OFFSET' => $nblogsnow,
         ]);
 
-        $this->integer(count($logs))->isIdenticalTo(45);
+        $this->integer(count($logs))->isIdenticalTo(47);
 
         $expected_types_count = [
             \Log::HISTORY_DELETE_SUBITEM => 5,//networkport and networkname
-            \Log::HISTORY_CREATE_ITEM => 16, //virtual machines, os, manufacturer, net ports, net names, ...
+            \Log::HISTORY_ADD_RELATION => 1, //new IPNetwork/IPAddress
```

Comment:
Now, on update, a new IPNetwork/IPAddress item is created.

It corresponds to:
```json
            {
                "description": "enp0s25",
                "driver": "e1000e",
                "ipaddress": "192.168.1.103",
                "ipgateway": "192.168.1.1",
                "ipmask": "255.255.255.0",
                "ipsubnet": "192.168.1.0",
                "mac": "b8:6b:23:e0:5c:81",
                "pciid": "8086:1559:1179:0001",
                "pcislot": "0000:00:19.0",
                "speed": "100",
                "status": "up",
                "type": "ethernet",
                "virtualdev": false
            },
```

See https://raw.githubusercontent.com/glpi-project/inventory_format/master/examples/computer_3_updated.json